bundles/sirix-core/src/main/java/org/sirix/utils/IntToObjectMap.java

```diff
@@ -151,7 +164,7 @@ public void remove() {
   private static int defaultCapacity = 16;
 
   /**
-   * Holds the base hash entries. if the capacity is 2^N, than the base hash
+   * Holds the base hash entries. if the capacity is 2^N, thn the base hash
```

Comment:
sill minor typo: I guess it's "then" ;-)

Reply:
Two times the charm :')